metadata-db-app/src/cmr/metadata_db/data/oracle/granule_table.clj

```diff
@@ -65,8 +55,9 @@
           table-name
           table-name))
 
-(defmethod granule-constraint-sql true
-  [provider table-name]
+(defn- granule-constraint-sql-true
```

Comment:
Is there a name without a boolean value in it that would work well?

Reply:
yes this function is now called create-granule-constraint-sql since this function is for the normal providers.

Reply:
I refactored the calling function and this function no longer exists.